src/docs/cookbook/forms/focus.md

```diff
@@ -33,15 +35,17 @@ TextField(
 );
 ```
 
-For more information on handling input and creating text fields, please see the
-[Forms section of the cookbook](/docs/cookbook#forms).
+For more information on handling input and creating text fields,
+see the [Forms](/docs/cookbook#forms) section of the cookbook.
```

Comment:
Other places you've used Cookbook (capitalized). (I rather like it uncapitalized.)